### PR TITLE
[FIND] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: base/applications/cmdutils/find/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /*
- * FILE:       base/applications/cmdutils/find/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Fulea Ștefan (PM on ReactOS Forum at fulea.stefan)
- * CHANGE LOG: 2011-08-20  initial translation
- *             2011-10-17  diacritics change, other minor changes
+ * PROJECT:     ReactOS Command Utils
```
Comment (binarymaster) [inline base/applications/cmdutils/find/lang/ro-RO.rc:2]:
```suggestion
 * PROJECT:     ReactOS Find Command
```

File: base/applications/cmdutils/find/lang/ro-RO.rc [hunk 2]
```diff
@@ -16,8 +16,8 @@ FIND [/V] [/C] [/N] [/I] [/OFF[LINE]] ""șir"" [fișier...]\n\
   /C         Numără liniile ce conțin șirul.\n\
   /N         Numără rândurile afișate, începând cu 1.\n\
   /I         Ignoră diferențele între majuscule și minuscule.\n\
-  /OFF[LINE] Do not skip files that have the offline attribute set."
+  /OFF[LINE] Nu sări fișierele care care au configurate atributul offline (deconectat)."
```
Comment (reactos-contributor) [inline base/applications/cmdutils/find/lang/ro-RO.rc:19]: "Omite" may also be an appropriate choice for translate "skip" in such context.

**Review by reactos-contributor — APPROVED** [review]

File: base/applications/cmdutils/find/lang/ro-RO.rc [hunk 3]
```diff
@@ -16,8 +16,8 @@ FIND [/V] [/C] [/N] [/I] [/OFF[LINE]] ""șir"" [fișier...]\n\
   /C         Numără liniile ce conțin șirul.\n\
   /N         Numără rândurile afișate, începând cu 1.\n\
   /I         Ignoră diferențele între majuscule și minuscule.\n\
-  /OFF[LINE] Do not skip files that have the offline attribute set."
+  /OFF[LINE] Nu omite fișierele care care au configurate atributul offline (deconectat)."
```
Comment (reactos-contributor) [inline base/applications/cmdutils/find/lang/ro-RO.rc:19]: This line is over 80 characters.

**Pull request merged**